OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ApplicationController.cs

```diff
@@ -303,6 +320,16 @@ public async Task<IActionResult> Create(ApplicationCreate applicationDto)
     [HttpPut]
     public async Task<IActionResult> Update(ApplicationUpdate applicationDto)
     {
+        if (await IsWorkshopOrProviderBlocked(applicationDto.WorkshopId).ConfigureAwait(false))
```

Comment:
(NRE) `applicationDto` might be null here. Check for null is already present in this method but it is performed later. Please change order of checks

Reply:
Done.